lib/src/bindings/bindings_player.dart

```diff
@@ -535,7 +591,45 @@ abstract class FlutterSoLoud {
 
   /// Set fader to oscillate the global volume at specified frequency.
   @mustBeOverridden
-  int oscillateGlobalVolume(double from, double to, Duration time);
+  PlayerErrors oscillateGlobalVolume(double from, double to, Duration time);
+
+  /// Fade a parameter of a filter.
+  ///
+  /// [handle] the handle of the voice to apply the fade. If equal to 0,
+  /// it fades the global filter.
+  /// [filterType] filter to modify a param.
+  /// [attributeId] the attribute index to fade.
+  /// [to] value the attribute should go in [time] duration.
+  /// [time] the fade slope duration.
+  /// Returns [PlayerErrors.noError] if no errors.
+  @mustBeOverridden
+  PlayerErrors fadeFilterParameter(
```

Comment:
These methods now accept also the `SoundHandle` or the `SoundHash`. These parameters are needed by the callers from `AudioSource` when managing filters. When called from `class SoLoud` these parameters are not needed because the filters are managed globally (for all the sounds).
In short, `AudioSource` manages filters for single sounds/voices and `SoLoud class` global filters.